app/models/validations/shared_validations.rb

```diff
@@ -108,4 +104,18 @@ def shared_validate_partner_count(record, max_people)
   def person_is_partner?(relationship)
     relationship == "P"
   end
+
+  def add_range_error(record, question)
+    field = question.check_answer_label || question.id
+    min = [question.prefix, number_with_delimiter(question.min, delimiter: ","), question.suffix].join("") if question.min
+    max = [question.prefix, number_with_delimiter(question.max, delimiter: ","), question.suffix].join("") if question.max
+
+    if min && max
+      record.errors.add question.id.to_sym, I18n.t("validations.numeric.within_range", field:, min:, max:)
+    elsif min
+      record.errors.add question.id.to_sym, I18n.t("validations.numeric.above_min", field:, min:)
+    else
+      record.errors.add question.id.to_sym, I18n.t("validations.numeric.below_max", field:, max:)
+    end
```

Comment:
I am not actually aware of a situation where there is a max but not a min after global searching `@max = ` and `"max": ` Is there any other way than those?
In which case I'm inclined to remove the else but keep the previous condition as elsif min
Agree?

Reply:
I don't think there's any other way - only possible exception I can think of are financial questions e.g. max salary that only display the max in an error message (rather than also saying min = 0). If that doesn't affect this then agree